packages/rOpenScPCA/R/calculate-clusters.R

```diff
@@ -6,6 +6,8 @@
 #' Specifically, the clustering algorithm defaults to "louvain" and the weighting scheme to "jaccard"
 #' to align with common practice in scRNA-seq analysis.
 #'
+#' @importFrom methods allNames
```

Comment:
probably safe to just do the whole thing here
```suggestion
#' @import methods
```